akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/StubbedActorContext.scala

```diff
@@ -179,10 +186,12 @@ private[akka] final class FunctionRef[-T](
    * Removal is asynchronous, explicit removeInbox is needed from outside afterwards.
    */
   override def stop[U](child: ActorRef[U]): Unit = {
-    if (child.path.parent != self.path) throw new IllegalArgumentException(
-      "Only direct children of an actor can be stopped through the actor context, " +
+    if (child.path.parent != self.path)
+      throw new IllegalArgumentException(
+        "Only direct children of an actor can be stopped through the actor context, " +
         s"but [$child] is not a child of [$self]. Stopping other actors has to be expressed as " +
-        "an explicit stop message that the actor accepts.")
+        "an explicit stop message that the actor accepts."
+      )
```

Comment:
Not a fan of the trailing new line for `)` can we configure that?

Reply:
I agree, fixed: `danglingParentheses = false`